File: solutions/0767-prime-number-of-set-bits-in-binary-representation/solution.py
class Solution(object):
    def countPrimeSetBits(self, left, right):
        """
        :type left: int
        :type right: int
        :rtype: int
        """
        def isprime(n):
            if n<2:
                return False
            for i in range(2, int(n ** 0.5) + 1):
                if n%i==0:
                    return False
            return True
        count=0
        for i in range(left,right+1):
            if isprime(bin(i).count('1')):
                count+=1
        return count

        

File: solutions/0767-prime-number-of-set-bits-in-binary-representation/solution.java
class Solution {
    public int countPrimeSetBits(int left, int right) {
    int count=0;
    for(int i=left;i<=right;i++)
    {
        int count1=0,a=i;
        while((a>0))
        {
            if((a&1)==1)
                count1++;
            a>>=1;
        }
        count+=santhu(count1);
    }
    return count;
    }
    int santhu(int count)
    {
        if(count == 2 || count == 3 || count == 5 || count == 7 ||
           count == 11 || count == 13 || count == 17 || count == 19)
            return 1;
        return 0;
    }
}
